src/constructN.js

```diff
@@ -24,7 +24,7 @@ var nAry = require('./nAry');
  *      };
  *      Salad.prototype.recipe = function() {
  *        var instructions = R.map((ingredient) => (
- *          'Add a whollop of ' + ingredient, this.ingredients)
+ *          'Add a whollop of ' + ingredient), this.ingredients
```

Comment:
1. Maybe to look nicer in REPL? Anyway one line is better, less confusion with the parenthesis.
2. Until that moment didn't even know what the "dollop" is, but sounds good for this example.

Also semicolons have some chaotic placement here, shall i add some?